src/Apps/Collect2/Routes/Collection/Components/Header/DefaultHeader.tsx

```diff
@@ -149,6 +155,11 @@ export const CollectionDefaultHeaderFragmentContainer = createFragmentContainer(
 const DefaultHeaderContainer = styled(Box)`
   background-color: ${color("black5")};
   overflow: hidden;
+
```

Comment:
I _think_ it should be declared before we don't want to apply this padding rule to then entire CollectionsApp component that `<HorizontalPadding>` wraps, rather it should only be applied to the Header component since we want to break the standard flow of the page by allowing the Header to extend beyond the entire Collections app's margins.

This rule is being applied to the single image header component as well: https://github.com/artsy/reaction/blob/7c8cab2985e7446dc03c3eb97f4f3dbc92c9da03/src/Apps/Collect2/Routes/Collection/Components/Header/index.tsx#L303

Reply:
The other reason I'm hesitant to make that change is that `HorizontalPadding` is a child of `FrameWithRecentlyViewed` which is being used in several places throughout the `Collect` and `Collections` pages In order to handle the conditional padding in the  `HorizontalPadding` component used in `FrameWithRecentlyViewed` then we'd have to pass a prop to indicate when to render padding there and when not to.

Reply:
Makes sense.